sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverClient.java

```diff
@@ -69,13 +72,12 @@ public String getEntityPath() {
     /**
      * Abandon a {@link ServiceBusReceivedMessage message} with its lock token. This will make the message available
      * again for processing. Abandoning a message will increase the delivery count on the message.
-     *
      * @param lockToken Lock token of the message.
      *
-     * @throws NullPointerException if {@code lockToken} is null.
+     * @throws NullPointerException          if {@code lockToken} is null.
      * @throws UnsupportedOperationException if the receiver was opened in {@link ReceiveMode#RECEIVE_AND_DELETE}
-     *     mode.
-     * @throws IllegalArgumentException if {@link MessageLockToken#getLockToken()} returns a null lock token.
+     *                                       mode.
```

Comment:
We don't align parameters.  You can change your IntelliJ Javadoc settings to not do this.